database/migrations/2018_10_23_203754_create_campaigns_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCampaignsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('campaigns', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('internal_title')->comment('Campaign title used internally.');
```

Comment:
@chloealee I saw elsewhere that you required this to be unique, should that be enforced here as well?

Reply:
@chloealee yes!

Reply:
@katiecrane oh great call ! Updated :)